test/bats/tests/constraints/all_ns_must_have_gatekeeper-dryrun.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: constraints.gatekeeper.sh/v1beta1
+kind: K8sRequiredLabels
+metadata:
+  name: ns-must-have-gk
```

Comment:
can we make this name a bit more unique? or i think `kubectl apply all_ns_must_have_gatekeeper.yaml` will override the dryrun one.

Reply:
that was my intention, so we can test update also. deploy with dry run, have it succeed, update with deny and have it deny.